lib/droplet_kit/resources/cdn_resource.rb

```diff
@@ -24,6 +24,12 @@ class CDNResource < ResourceKit::Resource
         handler(422) { |response| ErrorMapping.fail_with(FailedUpdate, response.body) }
       end
 
+      action :update_custom_domain, 'PUT /v2/cdn/endpoints/:id' do
+        body { |hash| { custom_domain: hash[:custom_domain] || '', certificate_id: hash[:certificate_id] || '' }.to_json }
```

Comment:
for readability, can you update to `hash.fetch(:custom_domain, '')`? Or even `hash[:custom_domain].to_s` which would convert the nil to an empty string if it isn't there.

Reply:
sure will do thank you.